lib/cufft/fft.jl

```diff
@@ -129,6 +139,19 @@ for f in (:fft, :bfft, :ifft)
 end
 rfft(x::DenseCuArray{<:Union{Integer,Rational}}, region=1:ndims(x)) = rfft(realfloat(x), region)
 plan_rfft(x::DenseCuArray{<:Real}, region) = plan_rfft(realfloat(x), region)
+irfft(x::DenseCuArray{<:Union{Real,Integer,Rational}}, d::Integer, region=1:ndims(x)) = irfft(complexfloat(x), d, region)
+
+# yields the maximal dimensions of the plan, for plans starting at dim 1 or ending at the size vector, this is always the full input size
+plan_max_dims(region, sz) = ifelse(region[1] == 1 && (length(region) <=1 || all(diff(collect(region)) .== 1)), length(sz), region[end])
```

Comment:
can you make these non single-line functions, both for legibility and because it breaks Coverage.jl?

Reply:
Sure. Will do.